### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by gabrielefronze**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -4,42 +4,35 @@ RUN yum search epel-release
...
 
-RUN echo 'export RUCIO_HOME="/home/rucio"' >> ~/.bashrc
-RUN echo 'export X509_USER_CERT="/home/rucio/etc/certs/ruciouser.pem"' >> ~/.bashrc
-RUN echo 'export X509_USER_KEY="/home/rucio/etc/certs/ruciouser.key.pem"' >> ~/.bashrc
+RUN ln -s /usr/bin/cmake3 /usr/bin/cmake
+RUN echo 'source scl_source enable devtoolset-8' >> ~/.bashrc
 
 COPY 5fca1cb1.0 /etc/grid-security/certificates/5fca1cb1.0
-COPY hostcert.pem /etc/grid-security
-COPY hostkey.pem /etc/grid-security
```
Comment (viveknigam3003) [inline Dockerfile:45]: I think we'll need this for `gfal-copy` to work

File: Dockerfile [hunk 2]
```diff
@@ -4,42 +4,35 @@ RUN yum search epel-release
...
 RUN pip3 install --upgrade pip setuptools wheel
 RUN pip3 install rucio-clients
-WORKDIR /home/rucio/etc
-RUN cp rucio.cfg.template rucio.cfg
-
-WORKDIR /home
 
-RUN echo 'export RUCIO_HOME="/home/rucio"' >> ~/.bashrc
-RUN echo 'export X509_USER_CERT="/home/rucio/etc/certs/ruciouser.pem"' >> ~/.bashrc
-RUN echo 'export X509_USER_KEY="/home/rucio/etc/certs/ruciouser.key.pem"' >> ~/.bashrc
```
Comment (viveknigam3003) [inline Dockerfile:41]: We cannot remove this as `gfal` looks for these env variables while initiating `gfal-copy`

**Review by viveknigam3003 — COMMENTED** [review]
Check out the comments

File: Dockerfile (hunk 2 above)
Comment (gabrielefronze) [reply Dockerfile:41]: For both comments: having the certs (and private keys!) distributed through the repository is a baaaad idea from multiple standpoints. :)
I'd suggest we opt for bind mounting the certs from the host instead.

Comment (viveknigam3003) [reply Dockerfile:41]: Ohh okay got it!

**Pull request merged**